examples/vanilla/advanced.html

```diff
@@ -31,13 +31,14 @@
   <body>
     <main>
       <h1>Media Chrome Advanced Video Usage Example</h1>
-      <media-controller defaultsubtitles>
+      <media-controller defaultsubtitles defaultduration="134">
```

Comment:
**NOTE:** we can revert this if preferred, though I think it might be a good example for `advanced`.